funcionalities/cart.py

```diff
@@ -0,0 +1,107 @@
+from utils import get_database_connection
+import json
+from models.classes import Product, Buyer
+from fastapi import HTTPException
+
+with open(r"data\upiti.json") as f:
+    file = json.load(f)
+
+
+def view_cart():
+    connection = get_database_connection()
+    cursor = connection.cursor()
+    cursor.execute(file["select_cart"])
+    results = cursor.fetchall()
+    cursor.close()
+    connection.close()
+    formatted_results = [
+        {
+            "korpa_id": row[0],
+            "kolicina": row[1],
+            "product_id": row[2],
+            "moj id": row[3],
+        }
+        for row in results
+    ]
+
+    return formatted_results
+
+
+def add_cart(kupac: Buyer, product: Product):
+    connection = get_database_connection()
+    cursor = connection.cursor()
+    params = (
+        kupac.kupac_id,
+        kupac.ime_kupca,
+        kupac.Prezime_kupca,
+        kupac.username,
+        kupac.password,
+    )
+    # proveravam da li je kupac registrovan
+    cursor.execute(file["select_kupci"], params)
+    res = cursor.fetchall()
+    if not res:
+        raise HTTPException(status_code=400, detail="buyer not found")
```

Comment:
This should be separated in log_in endpoint. You have this in view_cart_buyer as well. So the login check should happen separately. 
Create POST endpoint that is for log in